docs/migration-guide/to-15.md

```diff
@@ -165,3 +165,179 @@ If you would like to keep the previous behavior, you should change your config t
   ]
 }
 ```
+
+## List of Deprecated Rules
+
+These rules are deprecated - we removed them in v16. We recommend you use a pretty printer (like Prettier) alongside Stylelint rather than these rules.
```

Comment:
[suggest] I have minor suggestions:

- how about `x.y.z` format for consistency (e.g. `Migrating to 15.0.0`)?
- how about removing the details because Prettier is mentioned in the section above?

```suggestion
These rules are deprecated - we removed them in 16.0.0.
```

Anyway, thanks for moving the deprecated list. 😊 👍🏼